src/polus/plugins/_plugins/classes/plugin_classes.py

```diff
@@ -380,13 +378,40 @@ def submit_plugin(
 
     # Save the manifest if it doesn't already exist in the database
     organization = plugin.containerId.split("/")[0]
-    org_path = PLUGIN_DIR.joinpath(organization.lower())
+    org_path = _PLUGIN_DIR.joinpath(organization.lower())
     org_path.mkdir(exist_ok=True, parents=True)
     if not org_path.joinpath(out_name).exists():
-        with open(org_path.joinpath(out_name), "w") as fw:
-            json.dump(plugin.dict(), fw, indent=4)
+        with open(org_path.joinpath(out_name), "w", encoding="utf-8") as file:
+            manifest_ = plugin.dict()
+            manifest_["version"] = manifest_["version"]["version"]
+            json.dump(manifest_, file, indent=4)
 
-    # Refresh plugins list if refresh = True
-    if refresh:
-        _r()
+    # Refresh plugins list
+    refresh()
     return plugin
+
+
+def remove_plugin(plugin: str, version: typing.Optional[str] = None):
+    """Remove plugin from the local database."""
+    if version is not None:
+        for plugin_version in PLUGINS[plugin]:
+            remove_plugin(plugin, plugin_version)
+    else:
+        if not isinstance(version, Version):
+            version_ = cast_version(version)
```

Comment:
should you use the cast and so on to call remove_plugin() and have separate case for removing all versions (a list of versions should be doable to)

Reply:
Solved. Added support for lists of versions